src/amo/components/AddonMeta/index.js

```diff
@@ -58,8 +60,22 @@ export class AddonMetaBase extends React.Component<Props> {
               title: userTitle,
             },
             {
-              content: reviewCount,
-              title: reviewTitle,
+              content: (
+                <Link
+                  className="Addon-all-reviews-link"
+                  to={`/addon/${addon.slug}/reviews/`}
```

Comment:
This is generating an error when `addon` is null, because then one cannot access `addon.slug`. This is the error you were also seeing on Travis.

I think a nice way to deal with this would be to define a function which would create all of the links for you, because all of the link code is the same except for the content, so something like this:

```js
    const makeRatingLink = (content) => {
      if (addon) {
        return (
          <Link
            className="Addon-all-reviews-link"
            to={`/addon/${addon.slug}/reviews/`}
          >
            {content}
          </Link>
        );
      }
      return content;
    };
```

And then you could replace the above (for example) with `content: makeRatingLink(reviewCount)`.

Reply:
Thanks for response !! Definitely appreciate the feedback. I have used your suggestions to implement this function and now the tests are passing.